arrow-libs/fx/arrow-fx-stm/src/test/kotlin/arrow/fx/stm/STMTest.kt

```diff
@@ -103,7 +103,7 @@ class STMTest : ArrowFxSpec(
       val v1 = TVar.new(0)
       val v2 = TVar.new(0)
       val v3 = TVar.new(0)
-      val f = ForkConnected {
+      val f = async {
```

Comment:
A more theoretical question why we can substitute `ForkConnected` with `async`?

Reply:
Both launch a coroutine on a  `CoroutineContext`, here the one inherited from the parent or `Dispatchers.Default` by default and expose the result with a `await/cancel` contract.

So they model the exact same use-case. Which is also the reason why `Fiber` and `ForkXXX` is deprecated in favor of `kotlinx.coroutines.async`.

Reply:
Thanks 🙌